src/FSharpVSPowerTools.Core/OpenDeclarationsGetter.fs

```diff
@@ -46,23 +46,22 @@ module OpenDeclarationGetter =
 
     let private getAutoOpenModules entities =
         entities 
-        |> List.filter (fun e -> 
+        |> List.fold( fun acc e -> 
```

Comment:
I think in order to preserve ordering, `foldBack` should be used instead of `fold`.

One could use `choose` instead of filter + map, but `choose` need quite many allocations for temporary option values.